src/app/app.routing.ts

```diff
@@ -27,10 +26,11 @@ import { StageCompleteComponent } from './stage-complete/stage-complete.componen
 import { DoctorchatComponent } from './meds-n-labels/activity-3/activity-3.component';
 import { UnlockedStageComponent } from './unlocked-stage/unlocked-stage.component';
 import { HowToPlayComponent } from './unlocked-stage/how-to-play/howtoplay.component';
-import { UserUpdateComponent } from './user-update/user-update.component';
```

Comment:
not required?

Reply:
Ya! It is not required as we'll never be routing to User Update component in router outlet.

Reply:
Okay